pywbemtools/pywbemcli/_cmd_class.py

```diff
@@ -1054,20 +1054,24 @@ def get_namespaces(context, namespaces, default_rtn=False):
 
       context context (:class:`ContextObj` provided with command)
 
-      namespaces (list of (:term: str)
+      namespaces (tuple  or :term:string)
         List of strings where each string is one or more namespace names.
-        Any single string can contain multiple namespaces by comma-separating
-        the namespace names
+        Any single string can contain one or multiple namespaces by
+        comma-separating the namespace names
 
-      default_rtn (:class:`py:bool`):
-        Boolean that determines default return if namespaces is None or []:
+      default_all_ns (:class:`py:bool`):
+        Boolean that determines return value if namespaces is None or []:
           True: Return all namespaces in server
-          False: Return None (default)
+          TODO
+          False: Return default namespace for current conn (default)
+          None: Return None if default_rtn is None or all namespaces if
```

Comment:
1. You renamed `default_rtn`.
2. If `default_rtn` really means to be `default_all_ns`, then the "if" condition of this sentence is always true.

Reply:
guess that is what is called circular thinking.  Cleaned up the definition and just dropped the None alternative since the default is False.

DONE